src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -13,50 +13,50 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
         var map = new PatchMap();
 
         // Predefined patches
-        Patch patch0 = new Patch("Pangonian vents", 0, GetBiomeTemplate("aavolcanic_vent"));
+        Patch patch0 = new Patch("Pangonian vents", 0, GetBiomeTemplate("aavolcanic_vent"), 2500, 3000);
         patch0.AddSpecies(defaultSpecies);
         patch0.ScreenCoordinates = new Vector2(100, 400);
```

Comment:
set the depth here similarly to the screencoordinates.
Can do something like `patch0.Depth[0] = a;` etc.